content/docs/operator-guide/create-a-stack.md

```diff
@@ -0,0 +1,330 @@
+  useradd --uid ${CNB_USER_ID} --gid ${CNB_GROUP_ID} -m -s /bin/bash cnb
+
+# ========== ADDED ===========
+# 4. Install common packages
+RUN apt-get update && \
+  apt-get install -y xz-utils ca-certificates && \
+  rm -rf /var/lib/apt/lists/*
+```
+
+That should be it for our base image! Let's verify that we can successfully build this image by running:
+
+```bash
+docker build . -t cnbs/sample-stack-base:bionic --target base
+```
+
+### Creating the run image
+
+Next up, we will be creating the run image. The run image is the base image for your runtime application environemnt.
+
+In order to create our run image all we need to do is to set the run image's `USER` to the user with `CNB_USER_ID`. Our final `Dockerfile` for the build image should look like - 
+
+
+```Dockerfile
+# 1. Set a common base
+FROM ubuntu:bionic as base
+
+# 2. Set required CNB information
+ARG cnb_uid=1000
+ARG cnb_gid=1000
+ARG stack_id="io.buildpacks.samples.stacks.bionic"
+ENV CNB_USER_ID=${cnb_uid}
+ENV CNB_GROUP_ID=${cnb_gid}
+ENV CNB_STACK_ID=${stack_id}
+LABEL io.buildpacks.stack.id=${stack_id}
+
+# 3. Create the user
+RUN groupadd cnb --gid ${CNB_GROUP_ID} && \
+  useradd --uid ${CNB_USER_ID} --gid ${CNB_GROUP_ID} -m -s /bin/bash cnb
+
+# 4. Install common packages
+RUN apt-get update && \
+  apt-get install -y xz-utils ca-certificates && \
+  rm -rf /var/lib/apt/lists/*
+
+# ========== ADDED ===========
+# 5. Start a new run stage
+FROM base as run
+
+# 6. Set user and group (as declared in base image)
+USER ${CNB_USER_ID}:${CNB_GROUP_ID}
+```
+
+That should be it for our run image! Let's verify that we can successfully build this image by running:
+
+```bash
+docker build . -t cnbs/sample-stack-run:bionic --target run
+```
+
+### Creating the build image
+
+Next up, we will be creating the build image. The build image is the base image for you builder and should contain any common build-time
+environment required by your buildpacks.
+
+
+#### Install build packages
+
+Let's modify the `Dockerfile` to look like - 
+
+```Dockerfile
+# 1. Set a common base
+FROM ubuntu:bionic as base
+
+# 2. Set required CNB information
+ARG cnb_uid=1000
+ARG cnb_gid=1000
+ARG stack_id="io.buildpacks.samples.stacks.bionic"
+ENV CNB_USER_ID=${cnb_uid}
+ENV CNB_GROUP_ID=${cnb_gid}
+ENV CNB_STACK_ID=${stack_id}
+LABEL io.buildpacks.stack.id=${stack_id}
+
+# 3. Create the user
+RUN groupadd cnb --gid ${CNB_GROUP_ID} && \
+  useradd --uid ${CNB_USER_ID} --gid ${CNB_GROUP_ID} -m -s /bin/bash cnb
+
+# 4. Install common packages
+RUN apt-get update && \
+  apt-get install -y xz-utils ca-certificates && \
+  rm -rf /var/lib/apt/lists/*
+
+# ========== ADDED ===========
+# 5. Start a new run stage
+FROM base as run
+
+# 6. Set user and group (as declared in base image)
+USER ${CNB_USER_ID}:${CNB_GROUP_ID}
+
+# ========== ADDED ===========
+# 7. Start a new build stage
+FROM base as build
+
+# 8. Install packages that we want to make available at build time
+RUN apt-get update && \
```

Comment:
Modified this a little from our samples repo to make it self contained and avoid using COPY